SSOT/DeleteWorkExperienceIdFromOccupations.sql

```diff
@@ -0,0 +1,8 @@
+--Drop the foreign keys and WorkExperienceId column in Occupations table.
```

Comment:
Let's leave this script out of the batch file, because the backend code still expects `ExperienceId` even though the frontend no longer shows it.

Reply:
Ok, will not include it in bat file